src/main/java/net/snowflake/ingest/streaming/internal/OpenChannelResponse.java

```diff
@@ -21,6 +21,7 @@ class OpenChannelResponse extends StreamingIngestResponse {
   private List<ColumnMetadata> tableColumns;
   private String encryptionKey;
   private Long encryptionKeyId;
+  private StageMetadata stageLocation;
```

Comment:
the location metadata for internal stage vs. external volume should be the same, no? Maybe we should call it FileLocationInfo instead of StageMetadata so there's no stage vs. volume confusion in client sdk?

Reply:
it's good as long as we remove stage in the name

Reply:
Renamed.